docker/config/application.yaml

```diff
@@ -18,11 +18,11 @@
 # of these options.
 # This config is meant to be used by `compose-controller-spark-sql.yaml`.
 fhirdata:
-  fhirFetchMode: "HAPI_JDBC"
+  fhirFetchMode: "FHIR_SEARCH"
   # 172.17.0.1 is an example docker network interface ip address;
   # `hapi-server` is another docker example where a container with that name is
   # running on the same docker network.
-  # fhirServerUrl: "http://172.17.0.1:8091/fhir"
+  fhirServerUrl: "http://172.17.0.1:8091/fhir"
   # fhirServerUrl: "http://hapi-server:8080/fhir"
   dbConfig: "config/hapi-postgres-config_local.json"
```

Comment:
I think this should be commented out given that you have enabled `FHIR_SEARCH`.

Reply:
Done